gensim/topic_coherence/direct_confirmation_measure.py

```diff
@@ -37,7 +37,10 @@ def log_conditional_probability(segmented_topics, per_topic_postings, num_docs):
             w_prime_docs = per_topic_postings[w_prime]
             w_star_docs = per_topic_postings[w_star]
             co_docs = w_prime_docs.intersection(w_star_docs)
-            m_lc_i = np.log(((len(co_docs) / float(num_docs)) + EPSILON) / (len(w_star_docs) / float(num_docs)))
+            if  w_star_docs:
+                m_lc_i = np.log(((len(co_docs) / float(num_docs)) + EPSILON) / (len(w_star_docs) / float(num_docs)))
+            else:
+                m_lc_i = 0
```

Comment:
`0.0` better (make the type explicit).

Mixing floats and ints can lead to nasty bugs in Python.